package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "fliplet-widget-slider",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "devDependencies": {
+    "eslint-config-fliplet": "github:YaroslavOvdii/FlipletEsLint"
```

Comment:
this can now connect to `npm` https://www.npmjs.com/package/eslint-config-fliplet

Reply:
@squallstar 
installed a new npm module.